scripts/loadSponsorData.js

```diff
@@ -3,14 +3,21 @@
 const axios = require('axios');
 const config = require('../.config');
 const fs = require('fs');
-const mongoose = require('../');
+const mongoose = require('../types');
```

Comment:
I don't think this change will work, since we use `mongoose.connect()` here.

Reply:
had also noticed this, i dont know why, but vscode seems to want to resolve it there, have fixed it